### PR TITLE
Admin crud hotel 2

### PR DESCRIPTION
**Pull request by ndviet2307**

### CODE REVIEW

**Comment by ndviet2307** [comment]
screenshot from 2018-09-27 11-08-44

**Comment by lyvanminh** [comment]
OK

**Comment by ndviet2307** [comment]
ready

File: app/assets/javascripts/admin/hotel.js [hunk 1]
```diff
@@ -16,6 +16,43 @@ $(document).ready(function () {
...
+  });
+
+  $('.field:not(:first)').remove();
 });
+
+$(document).on('turbolinks:before-cache', function () {
+  if ($('#dataTables-hotel').DataTable()) {
+    $('#dataTables-hotel').DataTable().destroy();
+  }
+});
```
Comment (thanhhungchu95) [inline app/assets/javascripts/admin/hotel.js]: thêm dòng trống cuối file

**Comment by thanhhungchu95** [comment]
commented

**Comment by ndviet2307** [comment]
screenshot from 2018-09-27 13-23-10

**Comment by ngocphat301** [comment]
OK

**Comment by ndviet2307** [comment]
ready

**Pull request merged**